share/spice/biblio_search/books_on_personality/biblio_search_books_on_personality.js

```diff
@@ -0,0 +1,85 @@
+(function(env) {
+  "use strict";
+
+  env.ddg_spice_biblio_search_books_on_personality = function(api_result) {
+    console.info(api_result);
+    
+    String.prototype.trunc = String.prototype.trunc ||
+      function(n){
+          return (this.length > n) ? this.substr(0,n-1)+'&hellip;' : this;
+      };
+    
+    // Validating response
+    if(!api_result || api_result.GoodreadsResponse.search["total-results"].text == "0") {
+      return Spice.failed('biblio_search_books_by_authors');
+    }
+    
+    var personality = api_result.GoodreadsResponse.search.query.text,
+      source_url = "https://www.goodreads.com/search?q=" + personality + "&search_type=books";
+    
+    var get_details_uri = '/js/spice/biblio_search/get_book_details/',
+      get_image_for_book_uri = '/js/spice/biblio_search/get_image_for_book/';
+    
+    // Render the response
+    Spice.add({
+      id: "biblio_search_books_on_personality",
+      
+      // Customize these properties
+      name: "Books",
+      data: api_result.GoodreadsResponse.search.results.work,
+      meta: {
+        primaryText: "Books on " + personality,
+        sourceName: "GoodReads",
+        sourceUrl: source_url
+      },
+      onItemShown: function(item) {
+        $.ajaxSetup({ cache: true });
```

Comment:
No need to set this as each item shows. You can call the ajaxSetup once above and it should maintain it's setting